### ISSUE TITLE
Temporarily ignore accessibility test failures

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#3272]
## Description

In order for tests to pass on staging, we need to ignore accessibility test failures.
